web/src/controllers/Scripts.ts

```diff
@@ -145,6 +145,11 @@ export class Installers {
       response.status(404);
       return notFoundResponse;
     }
+
+    if (!kurlVersion && installer.spec.kurl) {
```

Comment:
@emosbaugh Here's where we use the custom spec that may have the installerVersion. I added this condition to check for the installerVersion in the spec if there wasn't one provided in the URL.